<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vyberte soubor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Soubor k podepsání:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.zip soubor s podpisem bude uložen do:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DirectoryPath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseDirectoryButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vyberte složku</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Podepiš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyFilesButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vygenerovat soubory s klíči</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseKeyFileButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vyberte soubor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyPath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Soukromý klíč pro zašifrování RSA:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPodpis">
    <property name="title">
     <string>Podpis</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOvereni">
    <property name="title">
     <string>Ověření</string>
    </property>
   </widget>
   <addaction name="menuPodpis"/>
   <addaction name="menuOvereni"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
